docs/events/eboard-meetings.rst

```diff
@@ -0,0 +1,49 @@
+########
+Eboard Meetings
+########
+
+This page documents RITlug eboard meetings.
+Topics include room booking, preparation, and running of the meeting.
+
+
+*****************
+Meeting checklist
+*****************
+
+Pre-meeting
+===========
+
+The President will triage the `RITlug operations board <https://github.com/orgs/RITlug/projects/1>`__, looking for tasks that require attention.
+The label **review at meeting** should be assigned to tasks that need explicit review by eboard.
+
+Meeting
+=======
+
+#. Designate a notetaker, typically the `secretary <../roles/secretary>`__, to record discussion points and action items for each task.
+   The notetaker should record their notes in the comments section of the task.
+
+#. A member will lead the meeting by reviewing the tasks assigned the **review at meeting** `label <https://github.com/orgs/RITlug/projects/1?card_filter_query=label%3A%22review+at+meeting%22>`__.
+
+If a discussion item requires work, and follow-up at a later meeting, and there is not a corresponding task, one should be created to track the work within the `RITlug operations board <https://github.com/orgs/RITlug/projects/1>`__.
+
+
+*************
+Date and time
+*************
+
+At the start of the semester, a day of week and time should be decided upon by the eboard members during which they are all free to meet on campus.
+Eboard members who are on co-op or otherwise unavailable for in-person meetings during the semester have the option to be videoconferenced into the meeting.
+
+Holding an eboard meeting several days ahead of the general meeting provides eboard the time to react to last-minute issues, such as speaker cancellations or content review.
+
+Currently the eboard meets on Monday. Historically the eboard has met the hour prior to general meetings.
```

Comment:
I'm wondering if it would be worthwhile to remove the first sentence, leaving the historically portion. If we leave the first sentence in, then this will have to be changed in the future to match when the current eboard meeting is taking place.

It may be a good idea to just move the "Historically.." sentence to the end of the previous paragraph.

Reply:
I would be okay to omit line 39 altogether. I think finding a time will depend on each eboard's schedules. It wasn't my first thought to mention our historical precedent of the hour prior because I personally found that meeting time to be difficult for us to plan since the meeting was immediately after. However, I'm okay with either outcome.

My only nitpick on this line is that it's two sentences on a single line, so if both are kept, they should be split into two lines.